Formula/pdftk-java.rb

```diff
@@ -0,0 +1,22 @@
+class PdftkJava < Formula
+  desc "Port of pdftk in java"
+  homepage "https://gitlab.com/pdftk-java/pdftk"
+  url "https://gitlab.com/pdftk-java/pdftk/-/archive/v3.0.6/pdftk-v3.0.6.tar.gz"
+  sha256 "bcc19573cd2cc316f8bd2039f01dd1b3b717405a25c58db589bde198bc63068b"
+
+  depends_on "gradle" => :build
+  depends_on :java => "1.7+"
+
+  def install
+    system "gradle", "installDist"
```

Comment:
Does this pull dependencies?

Reply:
good one, potentially yes. I guess I should try to use the binary jar package that they provide, and use `grade installDist` only for `head` installs